assets/locales/en.json

```diff
@@ -116,5 +117,14 @@
   "itemsFailedNotification": {
     "title": "Failed to add items",
     "message": "Could not add your items to your node."
+  },
+  "runGarbageCollectorWarning": {
+    "title": "Garbage collector",
+    "message": "Running the garbage collector will remove all the objects that are not pinned or on your MFS from your repository. This action might take a while. Do you want to run the Garbage Collector?",
+    "action": "Run GC"
```

Comment:
Maybe just “Run” - dialog is clear enough and having to parse what “GC” means can be confusing

Reply:
Yes, just "Run" is much better! :D